README.md

```diff
@@ -22,26 +23,47 @@ Once installed you can simple add `pyatmo` to your Python 3 scripts by including
 
     import pyatmo
 
-### Note ###
+Note
+----
 
-The module requires a valid user account and a registered application. See usage.md for further information.
+The module requires a valid user account and a registered application. See [usage.md](./usage.md) for further information.
 Be aware that the module may stop working if Netatmo decides to change their API.
 
+Development
+-----------
+
+To add the pre-commit hook to your environment run:
 
-### Testing ###
+    pip install pre-commit flake8 mypy black isort
+    pre-commit install
+
+Testing
+-------
 
 To run the pytest testsuite you need to install the following dependencies:
 
     pip install pytest pytest-mock pytest-cov requests-mock freezegun
 
+`pyatmo` has to be installed in editable mode to be picked up by pytest. Simply run:
+
+    pip install -e .
```

Comment:
pyatmo is already configured to work with pipenv so install can be reduce to `pipenv install`and it's create a virtual environment with pyatmo in it.

Reply:
Ok, great. I have no experience with `pipenv` so I take a look asap.